docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+    roles: [Bot]
+
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token.
+    bot_name: robot
+
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=access --logins=root
+  ```
+  </TabItem>
+</Tabs>
+
+
+Machine ID may now be configured using the `tbot configure` command.
+
+First, create a basic configuration file using the following parameters:
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+
+  <ScopedBlock scope={["oss", "enterprise"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=(=presets.tokens.first=) \
+     --join-method=token \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=auth.example.com:3025
+  ```
+
+  </ScopedBlock>
+  <ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=(=presets.tokens.first=) \
+     --join-method=token \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=example.teleport.sh:443
+  ```
+
+  </ScopedBlock>
+
+  </TabItem>
+  <TabItem label="IAM Method">
+
+  <ScopedBlock scope={["oss", "enterprise"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=iam-token \
+     --join-method=iam \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=auth.example.com:3025
+  ```
+
+  </ScopedBlock>
+  <ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=iam-token \
+     --join-method=iam \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=example.teleport.sh:443
+  ```
+
+  </ScopedBlock>
+
+  </TabItem>
+</Tabs>
+
+Replace the following fields with values from your own cluster.
+
+<ScopedBlock scope={["cloud"]}>
+
+- `token` is the token output by the `tctl bots add` command or the name of your IAM method token.
+- `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.
+- `data-dir` is where Machine ID writes its private data, including its own short-lived renewable certificates. These should not be used by applications and tools.
+- `auth-server` is the address of your Teleport Cloud Proxy Server, for example `example.teleport.sh:443`.
+- `certificate-ttl` is the option that will result in creation of a host certificate with Machine ID. The default TTL is for one hour, meaning that all generated certificates will expire after one hour's time.
```

Comment:
This sentence is confusing to me. It sounds like if you don't specify a TTL then a host certificate is not created, but I don't believe that's true.

Reply:
Clarifying this